fix posting date

yurirocha15 opened the pull request:
Fixes the rllib contribution post's date and change the text based on feedback.

jiyoonlim123 reviewed (commented):
Did you change asset folder name?
(assets/images/2021-08-10-open-source-contributions-rllib)

anencore94 commented:
@yurirocha15  오타가 있었습니다. 이 PR 에서 함께 수정해주시면 좋을 것 같네요

`# Ml-Agents 내부적인 기본 포트를 마추기`

yurirocha15 commented:
> Did you change asset folder name?
> (assets/images/2021-08-10-open-source-contributions-rllib)

Yes, I updated it together with the links.

> @yurirocha15 오타가 있었습니다. 이 PR 에서 함께 수정해주시면 좋을 것 같네요
> 
> `# Ml-Agents 내부적인 기본 포트를 마추기` -> 맞추기/지정하기/설정하기 등등

Fixed!
